lib/cocoapods_plugin.rb

```diff
@@ -0,0 +1,19 @@
+require 'cocoapods'
+
+Pod::Installer
+  .prepend(Module.new do
+             def perform_post_install_actions
+               super
+
+               return unless plugins.key?('refinement')
+
+               unless Gem::Version.create(Pod::VERSION) >= Gem::Version.create('1.6.0')
+                 raise Pod::Informative, 'Refinement requires a CocoaPods version >= 1.6.0'
```

Comment:
nit remove "a"

Also why can't this check be enforced at the `gemspec` level? Is it because 1.7.0 is not out yet?

Reply:
It's because this gem doesn't require CocoaPods to work

Reply:
Ah yes makes sense.